packages/http-client/package.json

```diff
@@ -39,10 +39,13 @@
     "url": "https://github.com/actions/toolkit/issues"
   },
   "devDependencies": {
+    "@types/node": "20.7.1",
     "@types/tunnel": "0.0.3",
-    "proxy": "^1.0.1"
+    "proxy": "^2.1.1",
+    "@types/proxy": "^1.0.1"
   },
   "dependencies": {
-    "tunnel": "^0.0.6"
+    "tunnel": "^0.0.6",
+    "undici": "^5.25.4"
```

Comment:
@takost,

When updating today, I noticed that size of my action bloated by 15k lines because `undici` was included as a dependency.

Reply:
the same for me. I've raised a ticket for the issue https://github.com/actions/toolkit/issues/1621